src/main/java/note/NoteList.java

```diff
@@ -89,13 +96,15 @@ public void deleteAllNotes() {
         ui.printDeleteAllNoteMessage();
     }
 
-    public void processNote(SceneList sceneList, String userChoice) {
-        if (userChoice.equals("create")) {
+    public void processNote(SceneList sceneList, String userChoice) throws InvalidNoteCommandException {
+        if (userChoice.equals("1")) {
             createNoteProcess(sceneList);
-        } else if (userChoice.equals("open")) {
+        } else if (userChoice.equals("2")) {
             openNoteProcess();
-        } else {
+        } else if (userChoice.equals("3")) {
             deleteNoteProcess();
+        } else {
+            throw new InvalidNoteCommandException(INVALID_NOTE_COMMAND);
```

Comment:
Just a suggestion can change to switch (userChoice) and use case statements might look neater?